src/Raven.Server/Documents/Handlers/DocumentReplicationHandler.cs

```diff
@@ -15,13 +14,21 @@ namespace Raven.Server.Documents.Handlers
 {
     public class DocumentReplicationRequestHandler : DatabaseRequestHandler
     {
-        [RavenAction("/databases*/lastSentChangeVector", "GET", "@/databases/{databaseName: string}/lastSentChangeVector")]
-        public async Task GetLastSentChangeVector()
+        [RavenAction("/databases*/lastSentEtag", "GET", "@/databases/{databaseName: string}/lastSentEtag?srcDbId={databaseUniqueId:string}")]
+        public Task GetLastSentEtag()
         {
+            var srcDbId = Guid.Parse(GetQueryStringValueAndAssertIfSingleAndNotEmpty("srcDbId")[0]);
+
             DocumentsOperationContext context;
             using (ContextPool.AllocateOperationContext(out context))
             {
+                var serverChangeVector = Database.DocumentsStorage.GetChangeVector(context);
+                var vectorEntry = serverChangeVector.FirstOrDefault(x => x.DbId == srcDbId);
+                
+                //no need to write a document for transferring a single value
+                HttpContext.Response.Headers[Constants.LastEtagFieldName] = vectorEntry.Etag.ToString();
```

Comment:
We want to avoid using headers, there are too many restrictions there. And if we need to look them up in the browser / curl, we want this visible.